resources/todomvc/architecture-examples/preact/src/app/model.js

```diff
@@ -1,13 +1,21 @@
-function uuid() {
-    let uuid = "";
-    for (let i = 0; i < 32; i++) {
-        let random = (Math.random() * 16) | 0;
-        if (i === 8 || i === 12 || i === 16 || i === 20)
-            uuid += "-";
+// Borrowed from https://github.com/ai/nanoid/blob/3.0.2/non-secure/index.js
+// This alphabet uses `A-Za-z0-9_-` symbols. A genetic algorithm helped
+// optimize the gzip compression for this alphabet.
+const urlAlphabet = "ModuleSymbhasOwnPr-0123456789ABCDEFGHNRVfgctiUvz_KqYTJkLxpZXIjQW";
```

Comment:
The alphabet here probably doesn't matter all that much for us, but any reason we don't use the latest version 3.X ?
https://github.com/ai/nanoid/blob/3.3.5/non-secure/index.js

Reply:
updated